docs/user/contributing/style.md

```diff
@@ -34,16 +34,19 @@ _In alphabetical order, not order of importance._
 
 ### Admonitions
 
-This section is based on the [Red Hat supplementary style guide.](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions).
+This section is based on the [Red Hat supplementary style guide](https://redhat-documentation.github.io/supplementary-style-guide/#admonitions).
 
 Admonitions should draw the reader’s attention to certain information. Keep admonitions to a minimum, and avoid placing multiple admonitions close to one another. If multiple admonitions are necessary, restructure the information by moving the less-important statements into the flow of the main content. See the [Docusaurus documentation](https://docusaurus.io/docs/markdown-features/admonitions) to learn the correct syntax.
 
 | Type           | Description                                                                                                                                                                                                                          |
 | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | Note           | Additional guidance or advice that improves product configuration, performance, or supportability                                                                                                                                    |
-| Important      | Advisory information essential to the completion of a task or default configuration settings. Users must not disregard this information.                                                                                             |
-| Warning/Danger | Information about potential system damage, data loss, or a support-related issue if the user disregards this admonition. If available, offer information to avoid the problem in the future or state where to find more information. |
 | Tip            | Recommendations, suggestions, and alternative methods that might not be obvious. Tips are not essential to using the product.                                                                                                        |
+| Info           | General information about product that may be useful or relevant to user.           |                                                                                                 
```

Comment:
```suggestion
| Info           | General information about the topic at hand that may be useful or relevant to user.           |                                                                                                 
```

Reply:
Ok, will fix this tomorrow. Already winding down for today.

Reply:
You mind If I commit it myself then?

Reply:
No, feel free to do so.